src/qt/bitcoin.cpp

```diff
@@ -38,7 +38,6 @@
 #if defined(BITCOIN_NEED_QT_PLUGINS) && !defined(_BITCOIN_QT_PLUGINS_INCLUDED)
 #define _BITCOIN_QT_PLUGINS_INCLUDED
```

Comment:
I think this is a false positive.

Reply:
but line 41 and 50?

Reply:
If you look closely each of them are nested inside #if define statements with different matching conditions.

Reply:
ah, I missed that entirely.  I'll revert that one